### PR TITLE
[15.0][ADD] sale_stock_invoice_plan

### PR DESCRIPTION
**Pull request by yankinmax**
After we removed `sale_stock` dependency from `sale_invoice_plan` we need to split it to separate module depending on `sale_invoice_plan`.

Depends on:
- [x] https://github.com/OCA/sale-workflow/pull/1979

### CODE REVIEW

**Review by gurneyalex — APPROVED** [review]

File: sale_stock_invoice_plan/readme/HISTORY.rst [hunk 1]
```diff
@@ -0,0 +1,11 @@
+15.0.1.1.0 (2022-05-10)
```
Comment (gurneyalex) [inline sale_stock_invoice_plan/readme/HISTORY.rst]: need to rewrite this file, to clear history of the previous module

File: sale_stock_invoice_plan/readme/USAGE.rst [hunk 2]
```diff
@@ -0,0 +1,7 @@
+- Create new sales quotation as per normal process
```
Comment (gurneyalex) [inline sale_stock_invoice_plan/readme/USAGE.rst]: rewrite to refer to the usage of sale_invoice_plan

File: sale_stock_invoice_plan/readme/DESCRIPTION.rst [hunk 3]
```diff
@@ -0,0 +1,4 @@
+By standard feature, user can gradually create partial invoices, one by one.
+This module add ability to create invoices based on the predefined invoice plan,
+either all at once, or one by one.
+The plan support both advance invoice and installment invoices.
```
Comment (gurneyalex) [inline sale_stock_invoice_plan/readme/DESCRIPTION.rst:2]:
```suggestion
The plan support both advance invoice and installment invoices.

This module allows to use sale_invoice_plan when the stock module is also installed. 
```

File: sale_stock_invoice_plan/readme/CONTRIBUTORS.rst [hunk 4]
```diff
@@ -0,0 +1,4 @@
+* Kitti Upariphutthiphong. <kittiu@gmail.com> (http://ecosoft.co.th)
+* `Trobz <https://trobz.com>`_:
+
+    * Son Ho <sonhd@trobz.com>
```
Comment (gurneyalex) [inline sale_stock_invoice_plan/readme/CONTRIBUTORS.rst:4]: you probably can add your name in there if you want :)

File: sale_stock_invoice_plan/readme/CREDITS.rst [hunk 5]
```diff
@@ -0,0 +1 @@
+The migration of this module from 13.0 to 14.0 was financially supported by Camptocamp
```
Comment (gurneyalex) [inline sale_stock_invoice_plan/readme/CREDITS.rst]: remove the file

**Comment by gurneyalex** [comment]
@OCA/community-maintainers is there an issue in the configuration of the code coverage tool in this repository? I find it strange that it flags the tests as not being covered by the tests.

**Review by rousseldenis — APPROVED** [review]

**Comment by rousseldenis** [comment]
> @OCA/community-maintainers is there an issue in the configuration of the code coverage tool in this repository? I find it strange that it flags the tests as not being covered by the tests.

I think it's well configured.

You can see the lines not covered here: https://app.codecov.io/gh/OCA/sale-workflow/compare/2009/changes

**Pull request merged**